spec/lib/va_profile/person/service_spec.rb

```diff
@@ -12,37 +12,19 @@
     subject { described_class.new(user) }
 
     let(:user) { build(:user, :loa3) }
-    let(:person_response) do
-      if Flipper.enabled?(:va_v3_contact_information_service)
```

Comment:
No longer needed.  VAProfile::V2::ContactInformation::PersonTransactionResponse will be used with the new PersonV2 service

Reply:
Cassette path variable  no longer needed